### PR TITLE
fix: Apply mod_websocket_session_ip for prosody 0.12.

### PR DESCRIPTION
**Pull request by bgrozev**
- Ignore .swp files.
- fix: Apply mod_websocket_session_ip for prosody 0.12.

### CODE REVIEW

File: ansible/roles/prosody/files/mod_websocket_session_ip-0.12.patch [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Set the session.ip property before the session filters are initialized when using mod_websocket
+
+diff -r 9ed628635dc6 -r 4480f9f470ad core/sessionmanager.lua
+--- a/core/sessionmanager.lua	Thu Feb 09 22:57:32 2023 +0000
++++ b/core/sessionmanager.lua	Wed Feb 15 12:25:53 2023 -0600
+@@ -31,6 +31,7 @@
+ 
+ local function new_session(conn)
+ 	local session = sessionlib.new("c2s");
++    session.ip = conn.http_remote_ip or conn:ip();
```
Comment (damencho) [inline ansible/roles/prosody/files/mod_websocket_session_ip-0.12.patch:17]: We do not need this one, right?

Comment (damencho) [reply ansible/roles/prosody/files/mod_websocket_session_ip-0.12.patch:17]: setting the ip is done few lines below in  `sessionlib.set_conn(session, conn);`

Comment (damencho) [reply ansible/roles/prosody/files/mod_websocket_session_ip-0.12.patch:17]: This is the change we tested on the drained shard

Comment (bgrozev) [reply ansible/roles/prosody/files/mod_websocket_session_ip-0.12.patch:17]: Yes, I was porting the previous patch and didn't see it. I'll fix it

**Review by damencho — APPROVED** [review]

**Comment by bgrozev** [comment]
8c518d8 is tested and works

**Review by aaronkvanmeerten — APPROVED** [review]

**Pull request merged**